### PR TITLE
Focus the first input/textarea after duplicating a wizard row

### LINKED ISSUE #6560: Improve usability in repeated fields like `listWizard`
### Description

In 99,9999% cases, when you use a `listWizard` backend widget, you want different values in each row. Currently the workflow is:

1. fill in a row
2. click on image icon
3. in the duplicated `input` field: a) click and select, or b) click and `backspace` multiple times, or c) triple-click if you're an advanced user
4. *then* you're free to type the new value

I think the expected (and more efficient) behaviour for the majority of users would instead be to have the duplicated field empty, or have the text in it already selected:

Image on 2023-11-24 10 12 34 PM

### PR DESCRIPTION
**Pull request by leofeyer**
Implements #6560

### CODE REVIEW

File: core-bundle/assets/scripts/core.js [hunk 1]
```diff
@@ -1406,6 +1409,7 @@ window.Backend =
 								ntr.inject(tr, 'after');
 								addEventsTo(ntr);
 								makeSortable(tbody);
+                                ntr.getFirst('td').getFirst('input').select();
```
Comment (ausi) [inline core-bundle/assets/scripts/core.js]:
```suggestion
								ntr.getFirst('td').getFirst('input').select();
```

File: core-bundle/assets/scripts/core.js [hunk 2]
```diff
@@ -1502,6 +1506,7 @@ window.Backend =
...
```
Comment (ausi) [inline core-bundle/assets/scripts/core.js]:
```suggestion
								ntr.getFirst('td').getFirst('input').select();
```

**Review by ausi — DISMISSED** [review]

File: core-bundle/assets/scripts/core.js (hunk 1 above)
Comment (leofeyer) [reply core-bundle/assets/scripts/core.js]: Fixed in 6031cc5e604cf747b073b7ff7b763b3d7c2f99d5.

File: core-bundle/assets/scripts/core.js (hunk 2 above)
Comment (leofeyer) [reply core-bundle/assets/scripts/core.js]: Fixed in 6031cc5e604cf747b073b7ff7b763b3d7c2f99d5.

**Pull request merged**